resources/migrations/051_add_new_query_execution_table.yaml

```diff
@@ -0,0 +1,82 @@
+databaseChangeLog:
+  - changeSet:
+      id: 51
+      author: camsaul
+      changes:
+        - createTable:
+            tableName: query_execution
+            remarks: 'A log of executed queries, used for calculating historic execution times, auditing, and other purposes.'
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: hash
```

Comment:
is there a way to recover the query in question?

or is the notion that we'd just use the card id to look it up?